extensions/s3-extensions/src/main/java/io/druid/storage/s3/S3DataSegmentKiller.java

```diff
@@ -63,7 +63,7 @@ public void kill(DataSegment segment) throws SegmentLoadingException
       }
     }
     catch (ServiceException e) {
-      throw new SegmentLoadingException(e, "Couldn't kill segment[%s]", segment.getIdentifier());
+      throw new SegmentLoadingException(e, "Couldn't kill segment[%s]: [%s]", segment.getIdentifier(), e);
```

Comment:
E as cause only captures stack trace. E as parameters has a lot of info in its toString